pom.xml

```diff
@@ -29,6 +45,52 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <plugin>
+                <!-- Building an executable JAR -->
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <version>3.0.2</version>
+                <configuration>
+                    <archive>
+                        <manifest>
+                            <addClasspath>true</addClasspath>
+                            <classpathPrefix>lib/</classpathPrefix>
+                            <mainClass>ru.ayaz.ChatServer</mainClass>
+                        </manifest>
+                    </archive>
+                </configuration>
+            </plugin>
+
+            <plugin>
+                <!-- Executing JAR-->
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <version>1.2.1</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>java</goal>
```

Comment:
Yes, it is outside of default build cycle. I decided not to place this goal to deploy phase. To make this goal work you need to write mvn exec:java and dont forget to edit configurations like mainClass and etc in your pom.xml file before.